integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -335,16 +335,17 @@ private int readdirInternal(String path, long buff, long filter, long offset,
 
   @Override
   public int open(String path, FuseFileInfo fi) {
-    return AlluxioFuseUtils.call(LOG, () -> openInternal(path, fi), "Fuse.Open", "path=%s", path);
+    final int flags = fi.flags.get();
+    boolean overwrite = OpenFlags.valueOf(flags) == OpenFlags.O_WRONLY;
+    String methodName = overwrite ? "Fuse.OpenOverwrite" : "Fuse.Open";
+    return AlluxioFuseUtils.call(LOG, () -> openInternal(path, fi, overwrite),
+        methodName, "path=%s,flags=%s", path, Integer.toHexString(flags));
```

Comment:
```suggestion
    return AlluxioFuseUtils.call(LOG, () -> openInternal(path, fi, overwrite),
        methodName, "path=%s,flags=0x%x", path, flags);
```
to avoid calling `Integer.toHexString(flags))` which can be expensive

Reply:
Make sense!